app/AppComponents.scala

```diff
@@ -30,7 +30,7 @@ class AppComponents(context: Context, identity: AppIdentity)
 
     override def controllerComponents: ControllerComponents = AppComponents.this.controllerComponents
 
-    override def panDomainSettings: PanDomainAuthSettingsRefresher = new PanDomainAuthSettingsRefresher(
+    override val panDomainSettings: PanDomainAuthSettingsRefresher = new PanDomainAuthSettingsRefresher(
```

Comment:
Note that this is necessary due to:

* https://github.com/guardian/pan-domain-authentication/pull/155